ansible/deploy_mailrelay.yml

```diff
@@ -0,0 +1,27 @@
+- name: "Install postfix"
+  hosts: all
+  sudo: yes
+  tasks:
+    - package: name=postfix state=present
+
+- name: "mail relay deploy"
+  hosts: mailrelay
+  sudo: yes
+  tasks:
+    - name: "sets our relayhost"
+      lineinfile:
+        dest: /etc/postfix/main.cf
+        regexp: '^relayhost ='
+        line: "relayhost = {{ mailrelay }}"
```

Comment:
i'm actually moving this variable to the inventory and calling it "parent_mailrelay" which makes more sense since this variable is dependant and affects only the mailrelay host  i've modified the dev inventory to test this in my dev environment.